### DISCUSSION TITLE
VS code online

### DISCUSSION THREAD

**Discussion by ayakasanma** [1101-GS3073-C/Python-Community-GS3073-C#79] [General]
https://vscode.dev/
開發者現在已經可以造訪網站來使用輕量級的VS code，可以在不便安裝IDE的裝置上使用！無論是手機或是平板都能輕鬆成為開發工具~~

**Comment by chia-kai-chang** [comment]
vscode可以安裝在windows, mac, linux
但是不能在ipad上面使用
有了線上版，以後就可以在ipad上面開發程式了~

**Comment by boson13579** [comment]
其實vscode 網頁版還是沒有載下來的好用喔~比如寫c++一定會裝的 C++ Extension 網頁版就不能用，還是推薦大家載下來用方便很多

**Comment by xCk27x** [comment]
之前有試過 但感想是超級不好用🥲
我試了好久都連不上github repository
後來就果斷放棄了🤒

**Comment by bww0401** [comment]
這樣就很方便ㄟ，如果是在外面臨時需要用Vs code ，就可以直接用手機了
